projects/ama-sdk/src/lib/helpers/create-entries-names.ts

```diff
@@ -15,6 +15,7 @@
  * limitations under the License.
  */
 
+import { sanitizeString } from 'ama-sdk';
```

Comment:
You can't import inside ama-sdk from 'ama-sdk'... It will cause circular dependency when building. Please use relative path.